usr/share/rear/finalize/Debian/i386/170_rebuild_initramfs.sh

```diff
@@ -1,7 +1,10 @@
 # rebuild the initramfs if the drivers changed
 #
-# probably not required, but I prefer to rely on this information when it
-# is backed by udev
+# probably not required, but I prefer to rely on this information when it is backed by udev
+# FIXME: who is 'I'?
+# Perhaps Schlomo Schapiro or someone who made the "P2V patch from Heinlein Support"?
```

Comment:
This is really long time ago. udev back then was indeed optional on some distros. I would say that this changed and udev is not optional any more.